.travis.yml

```diff
@@ -7,6 +7,9 @@ cache:
     - $HOME/virtualenv/python2.7.9/lib/python2.7/site-packages
     - node_modules
 
+install: pip install -r axe_selenium_python/tests/requirements/tests.txt
+script: skip
```

Comment:
A question, not a nit. What does this line do?

Reply:
the install tells Travis where the dependency file is, the default is ./requirements.txt

The second line is to prevent Travis from running tests in the deploy stage (since tests have already been run)

Reply:
Beauty 👍